test/uk/gov/hmrc/plasticpackagingtax/returns/controllers/actions/AuthActionSpec.scala

```diff
@@ -156,7 +162,7 @@ class AuthActionSpec extends ControllerSpec with MetricsMocks {
       authorizedUser(user)
 
       await(createAuthAction().invokeBlock(authRequest(Headers(), user), okResponseGenerator))
-      metricsMock.defaultRegistry.timer("ppt.returns.upstream.auth.timer").getCount should be > 1L
+      metricsMock.defaultRegistry.timer("ppt.returns.upstream.auth.timer").getCount should be > 0L
```

Comment:
This looks like a was passing by luck / previous test run order assert; taken to mean, the time was called at least once.